x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_logistics/step_logistics.tsx

```diff
@@ -648,7 +647,7 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
             />
 
             {/* Only display callout if includeFeatureState is enabled and the snapshot was created by ES 7.12+ */}
```

Comment:
[nit] Would you mind updating the comment to provide more context on how the versionId is interpreted?